src/integration/data/deleteMany.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Pinecone, Index } from '../../index';
+import {
+  randomString,
+  createIndexIfDoesNotExist,
+  generateRecords,
+} from '../test-helpers';
+
+describe('deleteMany', () => {
+  const INDEX_NAME = 'ts-integration';
+  let pinecone: Pinecone, index: Index, ns: Index, namespace: string;
```

Comment:
**nit / suggestion:** Really minor, but maybe tweak the variable names here for readability? 

```suggestion
  let pinecone: Pinecone, index: Index, namespace: Index, namespaceStr: string;
```